### DISCUSSION TITLE
How to use Host Time-dependent projects in Standalone (Native)?

### DISCUSSION THREAD

**Discussion by fotisandstuff** [DISTRHO/Cardinal#345] [Q&A] [answered]
I installed Cardinal Standalone (Native) on Windows 10 x64. Is there any way to use modules like the "Random-Progress-Pluck-Rev" without having to rely on external software? For example, could a play/stop button and a BPM selector be implemented? Or is there already a function like that? Thanks!

**Comment by falkTX** [comment, marked as answer]
No such functionality yet, those map to nothing at the moment unless you are using JACK.
My idea is to add a few controls on the menu bar, in the empty space or perhaps in the engine menu, to control playback and BPM.
Basically providing fake values, but good enough on a standalone.

No concrete plans on when this happens though.
